time timestamp
int64 state
float64[] rectifypos
float64[] robotPos


